puppet/zulip/manifests/app_frontend_base.pp

```diff
@@ -137,7 +137,7 @@
     group  => 'zulip',
     mode   => '0755',
   }
-  file { '/etc/cron.d/email-mirror':
+  file { '/etc/cron.d/email-gateway':
     ensure => absent,
   }
```

Comment:
I don't understand this change -- it looks like we were previously deleting the old filename?

Reply:
I think what's going on here is that ``app_frontend`` wants to do the postfix config (since I see it imports ``postfix_localmail``) and due  to that we have the above in ``app_frontend_base`` to ensure the cron file isn't present (since it is meant for the alternative IMAP setup)? And for that intent, I think having up-to-date cron filename there would be appropriate.

But I may be reading the intent behind this wrong, I'm not too familiar with our puppet system.